style.css

```diff
@@ -98,11 +98,36 @@ form {
 .submit-merchant-edits,
 .edit-merchant,
 .view-merchant-items {
-  padding: 5px;
+  padding: 3px;
+  border-width: 1px;
   margin-right: 10px;
+  border-radius: 20px;
   cursor: pointer;
 }
 
+.submit:active,
```

Comment:
I like the incorporation of active state styling when those elements get clicked!